docker-compose.yml

```diff
@@ -20,8 +20,7 @@ services:
     #  - AWS_REGION=us-east-1
 
   app:
-    build:
-      context: app
+    build: app
```

Comment:
Why removing the `context` key?
If one want to use team edition or change UID/GID (as advised below) it will need to re-put the [`context`](https://docs.docker.com/compose/compose-file/#context) to use the [`args`](https://docs.docker.com/compose/compose-file/#args) key.

Reply:
Yeah good point. I changed it to follow the pattern from the other services. Also, somebody should have caught that the file permissions were changed from `644 -> 755`.